### DISCUSSION TITLE
Refining design mockups and best practices

### DISCUSSION THREAD

**Discussion by slouie2** [twilio-labs/paste#937] [Office hours] [answered]
I would like feedback on my designs and mockups
- Are they following DS conventions?
- How to build a better UI, and improve consistency and composition in mockups

Background: Support plans page in Console
Stage of design: high fidelity mockups
Link to mockups: [here](https://docs.google.com/document/d/1ShSgjMQ6QVcbfnriJTiuOVhbfw4JkL697LCIMXVWXPg/edit#heading=h.62zi5967j5xd)

**Comment by kate-porter** [comment]
Thanks @slouie2! We'll be sure to familiarize ourselves with the work before office hours tomorrow.

**Comment by corinne-nagel** [comment, marked as answer]
Hi @slouie2 , thanks for coming to office hours today!

Here are the links we discussed:
[Code sandbox](https://codesandbox.io/s/paste-starter-kit-rj7yy)
[Engineering Getting Started](https://paste.twilio.design/getting-started/engineering/)
[Design Getting Started](https://paste.twilio.design/getting-started/design/)
[Components](https://paste.twilio.design/components)

[UX Review](https://docs.google.com/document/d/1kulA98Gn1TWd-FV2RqAaheNvZTNQQopbNUku1aKDYes/edit)
[UX CVB](https://docs.google.com/document/d/13n9F9h3_DuaFKWvvDSGHjAI-AtgAOSKDZlORkNzKBzM/edit)
[Twilio Way](https://docs.google.com/document/d/1ryGPMSnk1t34NHlrriFl4Cai0dCJhMW7U39dASQAAaQ/edit)


I know this is a lot so feel free to come back to office hours to review designs with us once you've added the components. Let us know if you have any questions!

**Reply by serifluous** [reply]
And some links to Paste Abstract projects:

- [Components](https://share.goabstract.com/789f3202-9d1d-42ce-8949-94f5e1b653bd)
- [Tokens](https://share.goabstract.com/5fc534a1-07a0-4bc5-bcc9-516faaa4d130)

We'll look into getting you an Abstract seat, but for now, feel free to export the Sketch files you need and use them locally. There are some Abstract tutorials linked in the Design Getting Started page Corinne linked, and you _should_ be able to see projects happening across the company—let us know if you can't find the Growth project.
